compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/test/root_op_attribute.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: iree-opt --split-input-file --iree-gpu-test-target=gfx942 --pass-pipeline='builtin.module(iree-llvmgpu-select-lowering-strategy)' %s | FileCheck %s
```

Comment:
This kind of test should locate at where the pass is. I.e., https://github.com/iree-org/iree/tree/main/compiler/src/iree/compiler/Codegen/LLVMGPU/test

Reply:
done